Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -2,28 +2,62 @@
 using System.Collections.Generic;
 using System.Linq;
 using Neuro.Pathfinding.DataStructures;
+using Reactor.Utilities.Extensions;
 using UnityEngine;
 
 namespace Neuro.Pathfinding;
 
 public sealed class PathfindingHandler
 {
-    private const float GRID_DENSITY = 6f; // TODO: Fine-tune individual maps to optimize performance
-    private const int GRID_BASE_WIDTH = 100;
+    // TODO: Reset these in an un-initialize function
+    internal float gridDensity = 0;
+    internal int gridBaseWidth = 0;
 
-    private const int GRID_SIZE = (int)(GRID_BASE_WIDTH * GRID_DENSITY);
-    private const int GRID_LOWER_BOUNDS = GRID_SIZE / -2;
-    private const int GRID_UPPER_BOUNDS = GRID_SIZE / 2;
+    internal int gridSize = 0;
+    private int _gridLowerBounds = 0;
+    private int _gridUpperBounds = 0;
 
```

Comment:
This task might be out of reach
![image](https://user-images.githubusercontent.com/32238504/230753117-5f079847-47f0-44ed-91f0-34eea3266199.png)

Reply:
Guess there is more fine-tuning. Fun